### ISSUE TITLE
Deeplab：a question about the “global average pooling” operation

### ISSUE THREAD

**Issue by toyow** [tensorflow/models#3769]
![clipboard](https://user-images.githubusercontent.com/37426295/37971537-c3d52c20-3208-11e8-85a3-ce29c2c82e77.png)

After reading the paper “Rethinking Atrous Convolution for Semantic Image Segmentation”,I'm confused by the “global average pooling” operation within the ASPP。

The original text is described as follows：
“Specifically, we apply global average pooling on the last feature map of the model, feed the resulting image-level features to a 1 × 1 convolution with 256 filters (and batch normalization [38]), and then bilinearly upsample the feature to the desired spatial dimension.”

I guess the ”global average pooling” means  turn the multilayered feature map(256 layers?) into one-dimensional vector(i.e image-level features ),the feature points  of this vector are average of the feature map.
After feeding the resulting image-level features to a 1 × 1 convolution with 256 filters,do we still get a new one-dimensional vector?And how can we bilinearly upsample a one-dimensional vector to a two-dimensional feature map?

I hope to get the answer, thank you!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ginobilinie** [comment]
@toyow Do you get the answer? I'm also confused by it.

**Comment by HanboHo** [comment]
@toyow  I have the same question like you. I really get confused about it. Have you already get the answer of it ?

**Comment by tbhunderbird** [comment]
+1. An answer to this question would be really nice!

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
